docs/troubleshooting/management/ts-management.asciidoc

```diff
@@ -67,7 +67,7 @@ image::images/endpoints-transform-failed.png[Endpoints page with Required transf
 To restart a transform that’s not running:
```

Comment:
The screenshot above these steps (endpoints-transform-failed.png) looks outdated. It's still showing `endpoint security` in the Endpoints page description.

Reply:
Screenshot updated!